sktime/utils/plotting.py

```diff
@@ -11,7 +11,36 @@
 from sktime.utils.validation.forecasting import check_y
 
 
-def plot_series(*series, labels=None, markers=None):
+def check_pred_int(pred_int):
+    """helper function to check pred_int data type
+
+
+    pred_int: pd.DataFrame
+        Prediction intervals of series
+
+
+    Riases
+    -------
+    TypeError: when pred_int is not a pd.DataFrame
+    Exception: when the number of columns is less or more than 2
+        and column labels are not ['lower', 'upper']
+    """
+    _check_soft_dependencies("pandas")
+    import pandas as pd
+
+    if isinstance(pred_int, pd.DataFrame):
+        if pred_int.shape[1] == 2:
+            if not pred_int.columns.isin(["lower", "upper"]).all():
+                raise Exception(
```

Comment:
```suggestion
                raise ValueError(
```

Reply:
raising more specific exceptions is more helpful I think

Reply:
How about: `Both pred_int columns must be labeled 'upper' and 'lower'` or `The upper and lower column labels of pred_int are undefined.`

And for the column size: `The pred_int dataframe must have exactly two columns`